ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/versions/altair/BeaconStateSchemaAltair.java

```diff
@@ -73,6 +99,14 @@
         getChildSchema(CURRENT_EPOCH_PARTICIPATION_FIELD_INDEX);
   }
 
+  public SyncCommitteeSchema getCurrentSyncCommitteeSchema() {
+    return (SyncCommitteeSchema) getChildSchema(CURRENT_SYNC_COMMITTEE_FIELD_INDEX);
```

Comment:
If we pull by field name, the schema's can handle arbitrary field reorderings: https://github.com/ConsenSys/teku/blob/0098b0390f3622e2e6eeb0ddd71245c0438fabc7/ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/BeaconStateSchema.java#L55-L58

Not sure that we'll need it, but kind of nice to be flexible.

Reply:
Seems reasonable - I'd just copied the patter from `getPreviousEpochParticipationSchema` so have updated them all.